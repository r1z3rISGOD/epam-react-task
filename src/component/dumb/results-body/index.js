import { ResultsBody } from './results-body'
export { ResultsBody }
