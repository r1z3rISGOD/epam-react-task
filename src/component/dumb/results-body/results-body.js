"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ResultsBody = void 0;
var React = require("react");
var ResultsBody = function (_a) {
    var children = _a.children;
    return (React.createElement("div", { className: "results-body" }, children));
};
exports.ResultsBody = ResultsBody;
//# sourceMappingURL=results-body.js.map