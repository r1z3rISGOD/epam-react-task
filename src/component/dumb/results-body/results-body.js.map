{"version":3,"file":"results-body.js","sourceRoot":"","sources":["results-body.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAMvB,IAAM,WAAW,GAAgC,UAAC,EAAY;QAAV,QAAQ,cAAA;IACjE,OAAO,CACD,6BAAK,SAAS,EAAC,cAAc,IAC1B,QAAQ,CACL,CACX,CAAA;AACH,CAAC,CAAA;AANY,QAAA,WAAW,eAMvB"}