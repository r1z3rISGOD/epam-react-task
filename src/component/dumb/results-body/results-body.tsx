import * as React from 'react'

interface ResultsBodyProps {
    children: React.ReactNode
}

export const ResultsBody : React.FC<ResultsBodyProps> = ({ children }) => {
  return (
        <div className="results-body">
          {children}
        </div>
  )
}
