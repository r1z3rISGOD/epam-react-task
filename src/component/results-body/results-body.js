import React from "react";
import "./results-body.scss"

export default function ResultsBody() {
    return(
        <div className="results-body-container">
            <div className="results-body">No Films Found</div>
        </div>
    )
}
