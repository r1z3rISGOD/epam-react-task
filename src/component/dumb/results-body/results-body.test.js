import { ResultsBody } from './results-body.tsx'
import React from 'react'

it('results body', () => {
  const TextInputComponent = shallow(<ResultsBody children={() => {}}/>)
  expect(TextInputComponent).toMatchSnapshot()
})
