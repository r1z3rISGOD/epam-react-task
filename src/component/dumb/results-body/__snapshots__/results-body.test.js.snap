// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`results body 1`] = `
<div
  className="results-body"
>
  <Component />
</div>
`;
